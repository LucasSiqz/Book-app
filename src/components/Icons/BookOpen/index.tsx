const BookOpen = () => (
  <svg
    width="17"
    height="16"
    viewBox="0 0 17 16"
    fill="none"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      fillRule="evenodd"
      clipRule="evenodd"
      d="M0 0H4.8C6.56731 0 8 1.59188 8 3.55556V16C8 14.5272 6.92548 13.3333 5.6 13.3333H0V0Z"
      stroke="#CFCBD2"
      strokeWidth="2"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
    <path
      fillRule="evenodd"
      clipRule="evenodd"
      d="M16 0H11.2C9.43269 0 8 1.59188 8 3.55556V16C8 14.5272 9.07452 13.3333 10.4 13.3333H16V0Z"
      stroke="#CFCBD2"
      strokeWidth="2"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
  </svg>
)

export default BookOpen
