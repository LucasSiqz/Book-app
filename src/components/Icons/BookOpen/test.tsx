import { render } from 'utils/test-utils'

import BookOpen from '.'

describe('<BookOpenIcon />', () => {
  it('should render correctly', () => {
    const { container } = render(<BookOpen />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
