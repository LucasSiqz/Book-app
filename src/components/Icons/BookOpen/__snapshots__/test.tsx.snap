// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BookOpenIcon /> should render correctly 1`] = `
<svg
  fill="none"
  height="16"
  viewBox="0 0 17 16"
  width="17"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clip-rule="evenodd"
    d="M0 0H4.8C6.56731 0 8 1.59188 8 3.55556V16C8 14.5272 6.92548 13.3333 5.6 13.3333H0V0Z"
    fill-rule="evenodd"
    stroke="#CFCBD2"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
  <path
    clip-rule="evenodd"
    d="M16 0H11.2C9.43269 0 8 1.59188 8 3.55556V16C8 14.5272 9.07452 13.3333 10.4 13.3333H16V0Z"
    fill-rule="evenodd"
    stroke="#CFCBD2"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="2"
  />
</svg>
`
